twine-cert.txt

```diff
@@ -1,348 +1,432 @@
+en = Set as favourite
 tags = common
 [certificate_action_button_check_validity]
 de = Gültigkeit prüfen
+en = Check validity
 tags = common
 [[Screen: check validity]]
 [certificate_check_validity_title]
 de = Gültigkeit prüfen
+en = Check validity
 tags = common
 [certificate_check_validity_message]
 de = Prüfen Sie vorab, ob Ihre Zertifikate in verschiedenen Ländern gültig sind. Hierfür werden die geltenden Einreiseregeln des gewählten Reiselands berücksichtigt.
+en = Check beforehand if your certificates are valid in different countries. This tool checks them against the travel regulations applicable in your chosen destination country.
 tags = common
 [certificate_check_validity_selection_country]
 de = Land
+en = Country
 tags = common
 [certificate_check_validity_selection_country_title]
 de = Reiseland wählen
+en = Choose destination
 tags = common
 [IT]
 de = Italien
+en = Italy
 tags = common
 [LT]
 de = Litauen
+en = Lithuania
 tags = common
 [DK]
 de = Dänemark
+en = Denmark
 tags = common
 [GR]
 de = Griechenland
+en = Greece
 tags = common
 [CZ]
 de = Tschechien
+en = Czech Republic
 tags = common
 [HR]
 de = Kroatien
+en = Croatia
 tags = common
 [IS]
 de = Island
+en = Iceland
 tags = common
 [PT]
 de = Portugal
+en = Portugal
 tags = common
 [PL]
 de = Polen
+en = Poland
 tags = common
 [BE]
 de = Belgien
+en = Belgium
 tags = common
 [BG]
 de = Bulgarien
+en = Bulgaria
 tags = common
 [DE]
 de = Deutschland
+en = Germany
 tags = common
 [LU]
 de = Luxemburg
+en = Luxembourg
 tags = common
 [EE]
 de = Estland
+en = Estonia
 tags = common
 [CY]
 de = Zypern
+en = Cyprus
 tags = common
 [ES]
 de = Spanien
+en = Spain
 tags = common
 [NL]
 de = Niederlande
+en = Netherlands
 tags = common
 [AT]
 de = Österreich
+en = Austria
 tags = common
 [LV]
 de = Lettland
+en = Latvia
 tags = common
 [LI]
 de = Liechtenstein
+en = Liechtenstein
 tags = common
 [FI]
 de = Finnland
+en = Finland
 tags = common
 [SE]
 de = Schweden
+en = Sweden
 tags = common
 [SI]
 de = Slowenien
+en = Slovenia
 tags = common
 [RO]
 de = Rumänien
+en = Romania
 tags = common
 [NO]
 de = Norwegen
+en = Norway
 tags = common
 [SK]
 de = Slowakei
+en = Slovakia
 tags = common
 [FR]
 de = Frankreich
+en = France
 tags = common
 [MT]
 de = Malta
+en = Malta
 tags = common
 [HU]
 de = Ungarn
+en = Hungary
 tags = common
 [IE]
 de = Irland
+en = Ireland
 tags = common
 [CH]
 de = Schweiz
+en = Switzerland
 tags = common
 [certificate_check_validity_selection_country_note]
-de = Sie finden Ihr Reiseland nicht? In einem mehrstufigen Prozess werden nach und nach mehr Länder verfügbar gemacht. \n\nMehr Informationen finden Sie in den Häufigen Fragen und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
+de = Sie finden Ihr Reiseland nicht? In einem mehrstufigen Prozess werden nach und nach mehr Länder verfügbar gemacht. \n\nMehr Informationen finden Sie in den #Häufigen Fragen::Häufigen Fragen# und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
```

Comment:
Please do not use those links at the moment `#Häufigen Fragen::Häufigen Fragen#`